lib/utils/index.js

```diff
@@ -811,57 +815,18 @@ module.exports = {
   /**
    * Get all props by looking at all component's properties
    * @param {ObjectExpression} componentObject Object with component definition
-   * @return {(ComponentArrayProp | ComponentObjectProp)[]} Array of component props in format: [{key?: String, value?: ASTNode, node: ASTNod}]
+   * @return {(ComponentArrayProp | ComponentObjectProp | ComponentUnknownProp)[]} Array of component props in format: [{key?: String, value?: ASTNode, node: ASTNod}]
    */
   getComponentProps(componentObject) {
-    const propsNode = componentObject.properties.find(
-      /**
-       * @param {ESNode} p
-       * @returns {p is (Property & { key: Identifier & {name: 'props'}, value: ObjectExpression | ArrayExpression })}
-       */
-      (p) => {
-        return (
-          p.type === 'Property' &&
-          getStaticPropertyName(p) === 'props' &&
-          (p.value.type === 'ObjectExpression' ||
-            p.value.type === 'ArrayExpression')
-        )
-      }
-    )
-
-    if (!propsNode) {
-      return []
-    }
-
-    return getComponentPropsFromDefine(propsNode.value)
+    return getComponentPropsFromOptions(componentObject)
```

Comment:
This is the only statement in the `getComponentProps` function now. Can we rename it to `getComponentPropsFromOptions` (also everywhere it is used) and only export a reference to `getComponentPropsFromOptions`?

```js
module.exports = {
  // …
  getComponentPropsFromOptions,
  // …
}

function getComponentPropsFromOptions(…) { … }
```

Or rename `getComponentPropsFromOptions` to `getComponentProps` and export only a reference to it?

```js
module.exports = {
  // …
  getComponentProps,
  // …
}

function getComponentProps(…) { … }
```

Or don't rename anything, but export still only export a reference to it?

```js
module.exports = {
  // …
  getComponentProps: getComponentPropsFromOptions,
  // …
}

function getComponentPropsFromOptions(…) { … }
```

Reply:
Thank you for your review! I changed method name and to use the shorthand property.
However, for backward compatibility, it also defines a method with the old name.